docs/best_practices/general.rst

```diff
@@ -22,32 +27,37 @@ Group. In this case the instances must have unique names. If they are both equal
 
-Names are not for storing meta-data. If you need to place other data of the same neurodata_type, you will need to
-choose another name. Keep in mind that meta-data should not be stored solely in the name of objects. It is OK to name
-an object something like “ElectricalSeries_large_array” however the name alone is not sufficient documentation. In this
-case, the source of the signal will be clear from the device of the rows from the linked electrodes table region, and
-you should also include any important distinguishing information in the description field of the object. Make an effort
-to make meta-data as explicit as possible. Good names help users but do not help applications parse your file.
 
-’/’ is not allowed in names. When creating a custom name, using the forward slash (/) is not allowed, as this can
-confuse h5py and lead to the creation of an additional group. Instead of including a forward slash in the name, please
-use “Over” like in DfOverF.
 
+.. _best_practice_name_slashes:
+
+Do Not Use Slashes
+~~~~~~~~~~~~~~~~~~
+
+'/' is not allowed in Group or Dataset names, as this can confuse h5py and lead to the creation of an additional group.
+Instead of including a forward slash in the name, please use "Over" like in DfOverF.
 
-Processing Modules
-------------------
+Check function: :py:meth:`~nwbinspector.checks.general.check_name_slashes`
 
 
-Indicate Modality
-~~~~~~~~~~~~~~~~~
 
-Give preference to default processing module names. These names mirror the extension module names: “ecephys”,
-“icephys”, “behavior”, “ophys”, “misc”.
+Descriptions
+------------
 
-We encourage the use of these defaults, but there may be some cases when deviating from this pattern is appropriate.
 
-For instance, if there is a processing step that involves data from multiple modalities, or if the user wants to
-compare two processing pipelines for a single modality (e.g. different spike sorters), you should create
-Processing Modules with custom names.
 
-ProcessingModules are themselves neurodata_types, and the other rules for neurodata_types also apply here.
+.. _best_practice_description:
+
+Metadata and Descriptions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Names are not for storing meta-data. If you need to place other data of the same neurodata_type, you will need to
+choose another name. Keep in mind that meta-data should not be stored solely in the name of objects. It is OK to name
+an object something like “ElectricalSeries_large_array” however the name alone is not sufficient documentation. In this
+case, the source of the signal will be clear from the device of the rows from the linked electrodes table region, and
+you should also include any important distinguishing information in the description field of the object. Make an effort
+to make meta-data as explicit as possible. Good names help users but do not help applications parse your file.
+
+As such, it is not recommended to use blank or default 'placeholder' descriptions.
```

Comment:
It seems like this section is touching on 3 separate issues:
* Storing metadata in names
* Using default names
* Good descriptions